### PR TITLE
Restructure directories. Move stateless components to their proper location.

### PR DESCRIPTION
**Pull request by jakereps**

### CODE REVIEW

File: app/js/components/Artifacts.jsx [hunk 1]
```diff
@@ -58,12 +52,13 @@ const Artifacts = ({ artifacts, dispatchDeleteArtifact }) => {
     );
 };
 
+Artifacts.contextTypes = {
```
Comment (ebolyen) [inline app/js/components/Artifacts.jsx]: Why is this necessary?

Comment (ebolyen) [inline app/js/components/Artifacts.jsx]: Looks like this is a leftover from [c6c87a0](https://github.com/qiime2/qiime-studio-frontend/pull/11/commits/c6c87a0e972d03c5c6deca2038a93be99cc28ea7)

File: app/js/containers/ArtifactsList.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+import Artifacts from '../components/Artifacts';
+import * as actionCreators from '../actions/Artifacts';
```
Comment (ebolyen) [inline app/js/containers/ArtifactsList.js]: Should this just be `import actions from '../actions';`?

**Comment by ebolyen** [comment]
I would make the files under `js/actions/` lowercase, the uppercase is mostly for React things.

File: app/js/components/Artifacts.jsx (hunk 1 above)
Comment (jakereps) [inline app/js/components/Artifacts.jsx]: This was from messing around with the redux `subscribe(listener)` where the example I was testing required using `store.getState()` and `store` is only accessible through `this.context`. It might eventually be necessary for showing loading indicators and such, but this was just accidentally left in. I'll cut it out.

File: app/js/containers/ArtifactsList.js (hunk 2 above)
Comment (jakereps) [inline app/js/containers/ArtifactsList.js]: I forgot to ask about that. Do we want to pull in the entirety of the action creators list (not really sure if it affects anything either way performance wise)? Then it should be that way for sure, but this was just pulling in the artifact specific actions.

**Comment by jakereps** [comment]
@ebolyen, just saw that this failed the build. It doesn't seem to have brought the case change along with the commit, even though my working directory has all lowercase filenames, and git status shows no difference. Working on it.

**Comment by ebolyen** [comment]
Looks like you fixed it! :+1:
